### PR TITLE
Xfl tracking

### PR DESCRIPTION
**Pull request by bcutler94**

### CODE REVIEW

File: uiengine/components/leaderboard/leaderboard.container.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+import { Leaderboard } from './leaderboard.component';
+import { ContestsNI } from '../../networkingResources/contests';
+
+const msp = (state = {}, {contestId}) => {
```
Comment (opetroulas) [inline uiengine/components/leaderboard/leaderboard.container.js]: eventually im going to force you to use mapState and mapDispatch like a civilized human

File: uiengine/components/scoreboard/scoreboard.container.js [hunk 2]
```diff
@@ -9,107 +9,30 @@ const failureHelper = () => {
...
-        homeTeamScore,
-        awayTeamScore,
-        homeTeamColor,
-        awayTeamColor,
-        gameClock,
-        gamePhase,
-        isFinal,
+        game,
+        homeTeam,
+        awayTeam,
```
Comment (opetroulas) [inline uiengine/components/scoreboard/scoreboard.container.js]: so clean

File: uiengine/components/trackingMyPicks/trackingMyPicks.container.js [hunk 3]
```diff
@@ -1,134 +1,38 @@
...
-        selections,
     }
 }
 
-const mdp = (dispatch, props) => {
+const mdp = (dispatch, {entry}) => {
+    const entryId = entry && entry._id
     return {
-        
+        getUserEntry: () => dispatch(EntriesNI.getUserEntry(entryId))
```
Comment (opetroulas) [inline uiengine/components/trackingMyPicks/trackingMyPicks.container.js]: don't you need to await this if you're going to show an error message / popup if loading fails or whatnot?

File: uiengine/networkingResources/entries.js [hunk 4]
```diff
@@ -1,27 +1,40 @@
...
-                }
+                    field: 'entry',
+                    defaultValue: {},
+                },
+
             ]
-        });
+        };
+
+        const { entry, error } = this.createGenericNetworkRequest(params);
```
Comment (opetroulas) [inline uiengine/networkingResources/entries.js]: you need to await createGenericNetworkRequest

File: uiengine/components/leaderboard/leaderboard.container.js (hunk 1 above)
Comment (bcutler94) [reply uiengine/components/leaderboard/leaderboard.container.js]: lol k noob

File: uiengine/components/scoreboard/scoreboard.container.js (hunk 2 above)
Comment (bcutler94) [reply uiengine/components/scoreboard/scoreboard.container.js]: so fresh

**Pull request merged**